src/components/main/compose/mod.rs

```diff
@@ -108,24 +108,24 @@ pub fn Compose(cx: Scope<Props>) -> Element {
                             }
 
                             // clicking the send button is meaningless if there isn't a conversation.
-                            if let Some(id) = current_chat {
+                            if let Some(convo) = current_chat.clone() {
 
                                 // mutate the state
-                                let cur = state.read().all_chats.get(&id).cloned();
+                                let cur = state.read().all_chats.get(&convo.conversation.id()).cloned();
```

Comment:
the following comment does't matter if `current_chat` gets changed back, but recall that `convo` should be the same as `cur` here.